server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -179,6 +180,7 @@ impl ChangelogTableName {
             ChangelogTableName::PackagingVariant => ChangeLogSyncStyle::Central,
             ChangelogTableName::IndicatorValue => ChangeLogSyncStyle::Legacy,
             ChangelogTableName::BundledItem => ChangeLogSyncStyle::Central,
+            ChangelogTableName::SystemLog => ChangeLogSyncStyle::Remote,
```

Comment:
I would suspect this would be 'special' type and in `create_filtered_outgoing_sync_query` we would add a special filter to system log and source site id ?